### ISSUE TITLE
[QUESTION] Will DeDRM work with the "Kindle for Windows 10" app?

### ISSUE THREAD

**Issue by l-l-l-l-l-l** [apprenticeharper/DeDRM_tools#1547]
I was looking for Kindle for PC 1.17 when I came accross a different app from Amazon called Kindle for Windows 10. Is it compatible with DeDRM?

**Comment by ElleKayEm** [comment]
I don't think so.  I'm not sure where its books are stored.  And I imagine DeDRM won't be able to find its key -- only looking for Kindle for PC stuff.
